src/codeflare_sdk/templates/base-template.yaml

```diff
@@ -42,6 +42,7 @@ spec:
         metadata:
           labels:
             workload.codeflare.dev/appwrapper: "aw-kuberay"
+            kueue.x-k8s.io/queue-name: local-queue
```

Comment:
Does this need to be added to the default template since we are just changing the name anyway?